tests/graph_tests.py

```diff
@@ -0,0 +1,35 @@
+from dsalgo.graph_traversal import Graph
+
+mat = [
+    [1, 3],
+    [1, 5],
```

Comment:
can you use unit tests here according to or guidelines?

Reply:
I'm not sure if unit test would work here, because BFS or DFS could have multiple correct answers.

Reply:
Add two or three tests of both dfs and bfs, you can take any or more of those multiple correct answers to work on.